src/components/tasks/TasksContent.tsx

```diff
@@ -144,16 +145,27 @@ export const TasksContent = ({ dev }: { dev?: boolean }) => {
 
     return (
         <div className={styles.tasksContainer}>
-            <TaskSearch
-                dev={dev}
-                onSelect={(selectedTab: Tab) =>
-                    searchNewTasks(selectedTab, queryAssignees, queryTitle)
-                }
-                inputValue={inputValue}
-                activeTab={selectedTab}
-                onInputChange={(value) => searchInputHandler(value)}
-                onClickSearchButton={searchButtonHandler}
-            />
+            {dev ? (
+                <TaskSearchDev
+                    onSelect={(selectedTab: Tab) =>
+                        searchNewTasks(selectedTab, queryAssignees, queryTitle)
+                    }
+                    inputValue={inputValue}
+                    activeTab={selectedTab}
+                    onInputChange={(value) => searchInputHandler(value)}
+                    onClickSearchButton={searchButtonHandler}
+                />
+            ) : (
+                <TaskSearch
+                    onSelect={(selectedTab: Tab) =>
+                        searchNewTasks(selectedTab, queryAssignees, queryTitle)
+                    }
+                    inputValue={inputValue}
+                    activeTab={selectedTab}
+                    onInputChange={(value) => searchInputHandler(value)}
+                    onClickSearchButton={searchButtonHandler}
+                />
```

Comment:
Why are we creating two separate components here?

Reply:
Because TaskSearch is becoming complex because of web accessibility & also it has to manage suggestions, userinput, etc alone. So it would be better if we separate the functionality to a different file along with the component itself based on dev flag.